include/exadg/structure/spatial_discretization/operators/elasticity_operator_base.cpp

```diff
@@ -41,12 +41,16 @@ ElasticityOperatorBase<dim, Number>::get_integrator_flags(bool const unsteady) c
 {
   IntegratorFlags flags;
 
-  flags.cell_evaluate  = CellFlags(unsteady, true, false);
-  flags.cell_integrate = CellFlags(unsteady, true, false);
+  dealii::EvaluationFlags::EvaluationFlags unsteady_flag;
```

Comment:
is it guaranteed that `unsteady_flag` does not contain any flag at this point, i.e. can we ommit an initialization with nothing? could still make sense for clarity in my opinion ...

Reply:
Resolved in dabca9637d3a61bb3f2860938613591bae2359e7